src/group_report.ts

```diff
@@ -156,15 +160,27 @@ export default class GroupReportGenerator {
                     case 'JVB':
                         // @TODO: convert JVB stats into more explict statuses
                         instanceReport.scaleStatus = 'ONLINE';
+                        if (instanceState.status.jvbStatus && instanceState.status.jvbStatus.participants) {
+                            instanceReport.scaleStatus = 'IN USE';
+                        }
+                        if (instanceState.status.jvbStatus && instanceState.status.jvbStatus.graceful_shutdown) {
```

Comment:
If instanceState.status.jvbStatus.graceful_shutdown is true, will the JVB also send instanceState.shutdownStatus?